.circleci/scripts/release-bump-manifest-version

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+if [[ "${CI:-}" != 'true' ]]
```

Comment:
I noticed the `${VAR:-}` parameter expansion being used throughout these scripts, and I'm not entirely sure what it does.

I am familiar with `${VAR:-foo}`, where `foo` would be substituted if `$VAR` was unset or null. But `${VAR:-}` would just substitute an empty string I suppose? Which seems irrelevant in all of the cases I've seen so far.

Reply:
Whoops, nevermind, I get it. It's because of `set -u`. I haven't used that in a while.

Reply:
Yeah I'm knowingly using a variable that might be unset and setting it to the empty string when that's the case. Since we're explicitly checking the value this isn't an issue.